squarelet/organizations/querysets.py

```diff
@@ -101,6 +101,15 @@ def get_viewable(self, user):
         else:
             return self.filter(plans__public=True)
 
+    def get_subscribed(self, user):
+        if user.is_authenticated:
+            return self.filter(
+                Q(plans__organizations__in=user.organizations.all())
+                | Q(client__owner=user)
```

Comment:
Does it make sense to include owned entitlements here?  I feel like maybe that should be separate?  Like one view to see all entitlements you are subscribed to and one to see all entitlements you own?